src/status_im/ui/screens/chat/message/reactions_row.cljs

```diff
@@ -2,22 +2,17 @@
   (:require [status-im.constants :as constants]
             [status-im.ui.screens.chat.message.styles :as styles]
             [quo.react-native :as rn]
-            [quo.core :as quo]))
+            [quo2.components.reactions.react :as quo2.react]))
 
-(defn reaction [{:keys [outgoing]} {:keys [own emoji-id quantity]} timeline]
-  [rn/view {:style (styles/reaction-style {:outgoing (and outgoing (not timeline))
-                                           :own      own})}
-   [rn/image {:source (get constants/reactions emoji-id)
-              :style  {:width        16
-                       :height       16
-                       :margin-right 4}}]
-   [quo/text {:accessibility-label (str "emoji-" emoji-id "-is-own-" own)
-              :style               (styles/reaction-quantity-style {:own own})}
-    quantity]])
-
-(defn message-reactions [message reactions timeline]
+(defn message-reactions [reactions timeline on-emoji-press on-open]
   (when (seq reactions)
-    [rn/view {:style (styles/reactions-row message timeline)}
-     (for [emoji-reaction reactions]
+    [rn/view {:style (styles/reactions-row timeline)}
+     (for [{:keys [own emoji-id quantity] :as emoji-reaction} reactions]
        ^{:key (str emoji-reaction)}
-       [reaction message emoji-reaction timeline])]))
+       [rn/view {:style {:margin-right 6 :margin-top 5}}
+        [quo2.react/render-react {:emoji (get constants/reactions emoji-id)
```

Comment:
probably better to rename both ns and component, to not confuse with react, something like `quo2.reactions/reaction`

Reply:
still not addressed